js_test.sh

```diff
@@ -1,5 +1,10 @@
 #!/bin/bash
 set -euf -o pipefail
 
-node ./node_modules/mocha/bin/_mocha --compilers js:babel-register static/js/global_init.js ${1:-'static/**/*/*_test.js'}
+node ./node_modules/mocha/bin/_mocha --compilers js:babel-register static/js/global_init.js ${1:-'static/**/*/*_test.js'} 2> >(tee npm-test.err)
```

Comment:
although it looks like a successful error-free test run will zero the file, so that the `-s` test will pass anyway?